_im-plugin/ism/policies.md

```diff
@@ -415,16 +415,36 @@ Parameter | Description | Type | Required | Default
 
 ### convert_index_to_remote
 
-Converts an index from a local snapshot repository to a remote repository.
+Converts an index to a searchable snapshot by restoring it from a remote snapshot repository. This action is useful for reducing storage costs by moving infrequently accessed data to remote storage while keeping it searchable. The action performs a restore operation from the specified snapshot and automatically deletes the original index once the restore is successfully accepted, ensuring only the remote index remains.
 
 The `convert_index_to_remote` operation has the following parameters.
 
 Parameter | Description | Type | Required | Default
 :--- | :--- |:--- |:--- |
-`repository` | The repository name registered through the native snapshot API operations.  | String | Yes | N/A
-`snapshot` | The snapshot name created through the snapshot action.  | String | Yes | N/A
+`repository` | The repository name registered through the native snapshot API operations. Must be a remote repository type (for example, S3, Azure, or GCS).  | `string` | Yes | N/A
```

Comment:
What are the actual valid values for this parameter? Is it `s3`, `azure`, and `gcs`? Please list them in code font.